### PR TITLE
Design apple home page

### PR DESCRIPTION
**Pull request by edemagbenyo**
Things learnt:
Position with flex and float
Backgroung: Image and gradient

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        </header>
+
+        <section class="intro">
+            <h1>What will your verse be?</h1>
+            <h2>Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.</h2>
+            <h3>Whatch the "Your Verse" film and explore the stories</h3>
+        </section>
+
+        <div class="featured">
+            <div><img src="./assets/img/1.jpg" alt=""></div>
```
Comment (wathigo) [inline index.html]: **[REQUIRED]** This images should be links to the articles or `"#"`. Please change the parent of the `image` element from `div` to `img`

**Review by wathigo — CHANGES_REQUESTED** [review]
### Great work @edemagbenyo  :clap: 
## The clone looks great!
**Please address the requirement below :point_down:  and submit another request for review**
_Happy Coding :)_

File: index.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                    <li>Support</li>
+                    <li>
+                        <form action="#">
+                            <input type="search" />
+                            <img src="./assets/img/search.png" alt="search icon">
+                        </form>
+                    </li>
+                </ul>
+            </nav>
+        </header>
```
Comment (Redvanisation) [inline index.html]: The `<header>` tag shouldn't be inside the `<main>` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                    <li>iPad</li>
+                    <li>iTunes</li>
+                    <li>Support</li>
+                    <li>
+                        <form action="#">
+                            <input type="search" />
+                            <img src="./assets/img/search.png" alt="search icon">
+                        </form>
+                    </li>
+                </ul>
```
Comment (Redvanisation) [inline index.html]: All of the `<li>` tags in the navigation should be links

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @edemagbenyo   🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. Please style the four bottom elements beneath the background picture just like on the original website.

2. Please consider my comments below.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @edemagbenyo 

**Well done implementing the changes requested, however the project doesn't look like it is required yet, please implement the changes below ⬇️**

- The page is still not looking like the original website, please fix the background to make it look exactly like the original apple page.

This is how it is looking on my screen: 

Screen Shot 2019-09-30 at 5 41 13 PM


### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by edemagbenyo** [comment]
We are probably looking at different copy of the same website. Kindly give me a link to your copy. This is what i am looking at https://web.archive.org/web/20140301004610/http://www.apple.com/.

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+  height: 50px;
+  border-radius: 5px;
+  background: linear-gradient(#7c807c 50%, #5f635f 50%);
+  color: #fff;
+  overflow: hidden;
+}
+
+nav ul li {
+  padding: 10px 30px;
+  border-left: 1px solid #fff;
```
Comment (bolah2009) [inline assets/css/style.css:51]: Why are we using white border left here? :thinking: Also, we could add hover effects as in the [Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/).

preffered apple nav hover

**Review by bolah2009 — APPROVED** [review]
## It's a :+1: from me

### [The clone](https://rawcdn.githack.com/edemagbenyo/apple-website/a433150ed2ecf2b78ee789a03e9925f94a47d8ae/index.html) uses CSS Backgrounds and griendeit effectively.

I would, however, suggest that the page to be longer (in height) than it is: Compare source page and the clone

|[Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/) (Preffered) | [Clone](https://rawcdn.githack.com/edemagbenyo/apple-website/a433150ed2ecf2b78ee789a03e9925f94a47d8ae/index.html) (Current)|
|:-----:|:-----:|
| Apple Source Page  | image  |

#### :heavy_check_mark: This project is approved :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**